package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Utilities for generating Bluemix artifacts",
   "main": "index.js",
   "engines": {
-    "node": ">=4"
+    "node": ">=6"
```

Comment:
We are usually bumping the minimal version to 8, because Node.js 6 is no longer supported. Is there any reason for keeping support for Node.js 6 in this project? If not, then let's use `>= 8.9` as we do in LB4.